windows/src/global/delphi/general/RegistryKeys.pas

```diff
@@ -263,6 +263,10 @@ interface
 
   SRegKey_ControlPanelInternationalUserProfile = 'Control Panel\International\User Profile'; // CU
```

Comment:
Do you a naming convention SRegKey_ControlPanelInternationalUserProfile_CU?

Reply:
It's a good idea -- it might be a way to improve all the key names in a future PR; the current 'design' is the only ones that have the `_CU`/`_LM` distinction are those that are in both CU/LM and differ (particularly relating to `Wow6432Node` and that kind of mumbojumbo)